cmd/network/verification.go

```diff
@@ -366,6 +372,16 @@ func (e *EgressVerification) generateAWSValidateEgressInput(ctx context.Context,
 	// Forward the timeout
 	input.Timeout = e.EgressTimeout
 
+	switch strings.ToLower(e.Probe) {
+	case "curl":
+		input.Probe = curl.Probe{}
+	case "legacy":
+		input.Probe = legacy.Probe{}
+	default:
+		fmt.Printf("Unrecognized probe %s - defaulting to curl probe.", e.Probe)
```

Comment:
```suggestion
		e.log.Info(ctx, "unrecognized probe %s - defaulting to curl probe.", e.Probe)
```
This command logs these kinds of messages